docs/Segments.md

```diff
@@ -118,6 +118,54 @@ This is platform specific; parses the data and interprets as a header for e.g. N
   start: 0xABC
 ```
 
+## `data`
+
+**Description:**
+
+Data located in the ROM.
+
+**Example:**
+
+```yaml
+# as list
+- [0xABC, data, filepath]
+
+# as dictionary
+- name: filepath
+  type: data
+  start: 0xABC
+```
+
+**Alternative:**
+
+If you want to migrate symbols from assembly to C, please prefix the `data` with a `.`, like `.data`. This instructs the linker to, in the build stage, link the `data` in the C file specified at `filepath`. 
+
+`splat` will not generate `.data.s` files for these sections, as the symbols should be declared in the specified C file.
+
+## `rodata`
+
+**Description:**
+
+Read-only data located in the ROM.
+
+**Example:**
+
+```yaml
+# as list
+- [0xABC, rodata, filepath]
+
+# as dictionary
+- name: filepath
+  type: rodata
+  start: 0xABC
+```
+
+**Alternative:**
+
+If you want to migrate symbols from assembly to C, please prefix the `rodata` with a `.`, like `.rodata`. This instructs the linker to, in the build stage, link the `data` in the C file specified at `filepath`. 
+
+`splat` will not generate `.rodata.s` files for these sections, as the symbols should be declared in the specified C file.
+
```

Comment:
`rodata` is a bit different than `data`. You usually want to figure out the asm file corresponding to a rodata section, then when you change an `asm` section to `C` you also want to change the corresponding `rodata` section to `.rodata` so the rodata gets migrated properly (floats, doubles, strings and jumptables)

Reply:
Does that mean `.data` should/can be migrated later? Any suggestions for how I can reword this so your sentiment is better reflected?

Reply:
The usual (and recommended) workflow is
- Do `c`/`.rodata` first
- Try to match functions and figure out types of other symbols
- Once you have figured out the types of the data section and you are confident about its file split then you migrate the data symbols into the C file (`data` -> `.data`)